### DISCUSSION TITLE
How can I install an extension?

### DISCUSSION THREAD

**Discussion by nhan000** [FreshRSS/Extensions#165] [Q&A] [answered]
Sorry I'm not a technical person so I don't understand the instructions in the Readme docs.

I'm using FreshRSS via flossboxin and via Edge browser. In the Configuration there is [ThreePanesView](https://framagit.org/nicofrand/xextension-threepanesview) but I don't know how to install it?
![image](https://github.com/FreshRSS/Extensions/assets/85216095/3cf93a19-4a85-43f7-97a9-998494535570)

It's not in this list so I can't just toggle it on
![image](https://github.com/FreshRSS/Extensions/assets/85216095/a948f614-95aa-458b-ba51-dd78eb0365da)

Thanks a lot for your help!

**Comment by math-GH** [comment, marked as answer]
You need access to the source code on your server. For more information, please read the manual: https://freshrss.github.io/FreshRSS/en/admins/15_extensions.html
